atom/browser/ui/views/menu_bar.cc

```diff
@@ -26,7 +26,7 @@ const SkColor kDefaultColor = SkColorSetARGB(255, 233, 233, 233);
 }  // namespace
 
 MenuBar::MenuBar(views::View* window)
-    : background_color_(kDefaultColor), menu_model_(NULL), window_(window) {
+    : background_color_(kDefaultColor), window_(window), menu_model_(NULL) {
```

Comment:
genuinely surprised that some of these, like order-of-initialization, weren't already warnings